packages/components/src/intl/intl/fr-CA.json

```diff
@@ -0,0 +1,3 @@
+{
+  "button.pending": "En attente"
```

Comment:
Why a capital letter here and not in English

Reply:
true, if anything, English often has the capital and french doesn't sometimes.